docker-compose.yml

```diff
@@ -50,4 +50,6 @@ services:
       DATABASE_URL: postgres://postgres:postgres@postgres:5432/jira-dev
       SQS_BACKFILL_QUEUE_URL: http://localstack:4566/000000000000/backfill
       SQS_PUSH_QUEUE_URL: http://localstack:4566/000000000000/push
+      SQS_DEPLOYMENT_QUEUE_URL: http://localstack:4566/000000000000/deployment
+      SQS_DISCOVERY_QUEUE_URL: http://localstack:4566/000000000000/discovery
```

Comment:
Had to add the discovery queue here because I had errors when testing locally.

Reply:
Good catch! Didn't notice that because I never run the app in docker for local testing